docs/installing_server.md

```diff
@@ -397,11 +415,11 @@ to each Rucio account which is meant to use the OIDC authN/Z.
 In case you wish to use OIDC by default in order to login to the Rucio
 WebUI, one has to configure also another block in the
 `rucio.cfg` file:
-
-    [webui]
-    auth_type = oidc
-    auth_issuer = <IdP nickname from the idpsecrets.json file>
-
+```bash
+  [webui]
+  auth_type = oidc
+  auth_issuer = <IdP nickname from the idpsecrets.json file>
+```
```

Comment:
This is not a bash syntax. This is a cfg file which can be highlighted as plain text